src/ModulesServiceProvider.php

```diff
@@ -9,12 +9,6 @@
 
 abstract class ModulesServiceProvider extends ServiceProvider
 {
-    /**
-     * Indicates if loading of the provider is deferred.
-     *
-     * @var bool
-     */
-    protected $defer = false;
```

Comment:
Was this removed in laravel?

Reply:
> Deferred Service Providers
Likelihood Of Impact: Medium

The defer boolean property on the service provider which is/was used to indicate if a provider is deferred has been deprecated. In order to mark the service provider as deferred it should implement the Illuminate\Contracts\Support\DeferrableProvider contract.

Reply:
Oh nice ok.